Hymns/Display/DisplayHymnView.swift

```diff
@@ -22,7 +22,10 @@ struct DisplayHymnView: View {
                 Text(viewModel.title)
                     .fontWeight(.semibold)
                 Spacer()
-                Image(systemName: "heart").padding()
+                Button(action: {self.viewModel.toggleFavorited()
+                }) {
```

Comment:
can you fix these lint warnings while you're at it

Reply:
Addressed in commit e9e8a89